ios/MissionHub-Bridging-Header.h

```diff
@@ -2,3 +2,10 @@
 //  Use this file to import your target's public headers that you would like to expose to Swift.
 //
 
+// The Google/Analytics pod has been deprecated. Since moving to the GoogleAnalytics pod, these need to be manually added to the bridging header.
+#import <GoogleAnalytics/GAI.h>
+#import <GoogleAnalytics/GAIFields.h>
+#import <GoogleAnalytics/GAILogger.h>
+#import <GoogleAnalytics/GAIDictionaryBuilder.h>
+#import <GoogleAnalytics/GAITrackedViewController.h>
+#import <GoogleAnalytics/GAITracker.h>
```

Comment:
So these are needed? Not sure I understand the comment. The Pod is deprecated but we recently moved to use it?

Reply:
@OzzieOrca Yeah I'm not too sure. I'm basing these changes on how it was done for GodTools on this pr https://github.com/CruGlobal/godtools-swift/pull/619